stylesheets/_layout.settings.scss

```diff
@@ -467,3 +467,101 @@
 .content-actions__buttons .btn {
     vertical-align: top;
 }
+
+/**
+ * Stacked inputs
+ */
+ .tab__settings .settings-section--stacked {
+
+    .form__group {
+        clear: both;
+    }
+
+    // Force labels to be stacked with the inputs below, except for checkbox/radio
+    .form__group:not(.form__group--flush):not(.form__group--toggle):not(.form-checkbox):not(.form-radio) {
+        margin-bottom: 12px;
+
+        > .control__label {
+            padding-top: 0;
+        }
+
+        > .control__label,
+        .controls,
+        .form__control:not(.checkbox):not(.radio),
+        .form__control + .select2,
+        .input-group {
+            width: 100%;
+        }
+
+        .ie9 &.colorpicker .form__control {
```

Comment:
Can we use `@include ie-lte(9)` or `@include ie(9)` here so as not to pollute the main stylesheet with ie9 overrides.